_posts/api/webpage/method/2000-01-01-switch-to-child-frame.md

```diff
@@ -14,6 +14,18 @@ Deprecated.
 ```javascript
 var webPage = require('webpage');
 var page = webPage.create();
+var url = "http://my.test.com";   // It has iframes;
```

Comment:
Better to use a real functioning site for an example. Otherwise, it will be confusing.